kitsune/sumo/parser.py

```diff
@@ -44,6 +44,8 @@
 }
 VIDEO_PARAMS = ["height", "width", "modal", "title", "placeholder"]
 YOUTUBE_PLACEHOLDER = "YOUTUBE_EMBED_PLACEHOLDER_%s"
+UI_COMPONENT_PLACEHOLDER = "UI_COMPONENT_EMBED_PLACEHOLDER_%s"
+ALLOWED_UI_COMPONENTS = frozenset(["device_migration_wizard"])
```

Comment:
why are you using a frozenset here? 
Also the idea of using the iterable is to allow multiple UI components in the future?

Reply:
> why are you using a frozenset here?
Just to indicate that it's immutable, that additional UI component names can't be added dynamically.
> Also the idea of using the iterable is to allow multiple UI components in the future?
Yes, just to remain open to that possibility.